rest_framework/relations.py

```diff
@@ -106,7 +108,14 @@ def get_attribute(self, instance):
             # Optimized case, return a mock object only containing the pk attribute.
             try:
                 instance = get_attribute(instance, self.source_attrs[:-1])
-                return PKOnlyObject(pk=instance.serializable_value(self.source_attrs[-1]))
+
```

Comment:
I think we could probably drop the extra whitespacing in this block (one line here)